eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -148,6 +148,9 @@ jobs:
       - name: clrRuntimePortableBuildArg
         value: '-portablebuild=false'
 
+    - name: CoreClrPgoDataArg
+      value: ''
+
```

Comment:
```suggestion
    # Set a default empty argument for the pgo path.
    # This will be set during the 'native prerequisites' step if PGO optimization is enabled.
    - name: CoreClrPgoDataArg
      value: ''

```